.gitignore

```diff
@@ -5,4 +5,7 @@
 # Packages
 *.egg
 *.egg-info
-dist
\ No newline at end of file
+dist
+
+# for sjr local dev
+Makefile
```

Comment:
This might be a candidate for a [global gitignore?](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) fine to keep it in.